packages/utils/src/lib/text-formats/constants.ts

```diff
@@ -0,0 +1,10 @@
+export const FONT_STYLES = {
+  italic: 'italic',
+  bold: 'bold',
+  'strike through': 'strike-through',
```

Comment:
Maybe we use "strikeThrough" or "strike-through" as key?
```suggestion
  'strikeThrough': 'strike-through',
```

Reply:
Why even have separate keys and values? How about this:

```ts
export const FONT_STYLES = ['bold', 'italic', 'strike-through'] as const;
```

You can still infer the type:

```ts
export type FontStyle = (typeof FONT_STYLES)[number];
```

Reply:
removed font style all at ones